<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>muu_summa</class>
 <widget class="QWidget" name="muu_summa">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>388</width>
    <height>414</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Form</string>
  </property>
  <widget class="QPushButton" name="no_button_9">
   <property name="geometry">
    <rect>
     <x>180</x>
     <y>290</y>
     <width>51</width>
     <height>51</height>
    </rect>
   </property>
   <property name="text">
    <string>9</string>
   </property>
  </widget>
  <widget class="QPushButton" name="no_button_5">
   <property name="geometry">
    <rect>
     <x>120</x>
     <y>230</y>
     <width>51</width>
     <height>51</height>
    </rect>
   </property>
   <property name="text">
    <string>5</string>
   </property>
  </widget>
  <widget class="QPushButton" name="no_button_8">
   <property name="geometry">
    <rect>
     <x>120</x>
     <y>290</y>
     <width>51</width>
     <height>51</height>
    </rect>
   </property>
   <property name="text">
    <string>8</string>
   </property>
  </widget>
  <widget class="QPushButton" name="no_button_4">
   <property name="geometry">
    <rect>
     <x>60</x>
     <y>230</y>
     <width>51</width>
     <height>51</height>
    </rect>
   </property>
   <property name="text">
    <string>4</string>
   </property>
  </widget>
  <widget class="QPushButton" name="no_button_2">
   <property name="geometry">
    <rect>
     <x>120</x>
     <y>170</y>
     <width>51</width>
     <height>51</height>
    </rect>
   </property>
   <property name="text">
    <string>2</string>
   </property>
  </widget>
  <widget class="QPushButton" name="no_button_3">
   <property name="geometry">
    <rect>
     <x>180</x>
     <y>170</y>
     <width>51</width>
     <height>51</height>
    </rect>
   </property>
   <property name="text">
    <string>3</string>
   </property>
  </widget>
  <widget class="QPushButton" name="no_button_7">
   <property name="geometry">
    <rect>
     <x>60</x>
     <y>290</y>
     <width>51</width>
     <height>51</height>
    </rect>
   </property>
   <property name="text">
    <string>7</string>
   </property>
  </widget>
  <widget class="QPushButton" name="no_button_1">
   <property name="geometry">
    <rect>
     <x>60</x>
     <y>170</y>
     <width>51</width>
     <height>51</height>
    </rect>
   </property>
   <property name="text">
    <string>1</string>
   </property>
  </widget>
  <widget class="QPushButton" name="no_button_6">
   <property name="geometry">
    <rect>
     <x>180</x>
     <y>230</y>
     <width>51</width>
     <height>51</height>
    </rect>
   </property>
   <property name="text">
    <string>6</string>
   </property>
  </widget>
  <widget class="QPushButton" name="no_button_BACK">
   <property name="geometry">
    <rect>
     <x>240</x>
     <y>170</y>
     <width>101</width>
     <height>51</height>
    </rect>
   </property>
   <property name="text">
    <string>BACK</string>
   </property>
  </widget>
  <widget class="QLineEdit" name="no_line_maara">
   <property name="geometry">
    <rect>
     <x>60</x>
     <y>110</y>
     <width>241</width>
     <height>41</height>
    </rect>
   </property>
  </widget>
  <widget class="QLabel" name="label">
   <property name="geometry">
    <rect>
     <x>60</x>
     <y>40</y>
     <width>191</width>
     <height>61</height>
    </rect>
   </property>
   <property name="font">
    <font>
     <pointsize>20</pointsize>
    </font>
   </property>
   <property name="text">
    <string>MUU SUMMA</string>
   </property>
  </widget>
  <widget class="QPushButton" name="no_button_0">
   <property name="geometry">
    <rect>
     <x>60</x>
     <y>350</y>
     <width>51</width>
     <height>51</height>
    </rect>
   </property>
   <property name="text">
    <string>0</string>
   </property>
  </widget>
  <widget class="QPushButton" name="no_button_hyvaksy">
   <property name="geometry">
    <rect>
     <x>240</x>
     <y>230</y>
     <width>101</width>
     <height>51</height>
    </rect>
   </property>
   <property name="text">
    <string>HYVÃ„KSY</string>
   </property>
  </widget>
  <widget class="QPushButton" name="no_button_takaisin">
   <property name="geometry">
    <rect>
     <x>240</x>
     <y>290</y>
     <width>101</width>
     <height>51</height>
    </rect>
   </property>
   <property name="text">
    <string>TAKAISIN</string>
   </property>
  </widget>
 </widget>
 <resources/>
 <connections/>
</ui>
